### PR TITLE
Remove deprecated plain text auth references for wodles

### LINKED ISSUE #6363: Setting up AWS credentials in `ossec.conf`
We need to revise the deprecated notice of the option to configure credentials in the `ossec.conf` file

image

and the showcasing of the feature in examples

image

image

image

image

### PR DESCRIPTION
**Pull request by nico-stefani**
## Description

This PR removes the plain text authentication references for AWS and Azure wodles configurations.

### Docs building
- [x] Compiles without warnings.

### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by GGP1 — APPROVED** [review]
LGTM

File: source/amazon/services/prerequisites/credentials.rst [hunk 1]
```diff
@@ -181,20 +180,3 @@ If you're using a single AWS account for all your buckets this could be the most
...
-
-This is an example configuration:
-
-.. code-block:: xml
-
-  <bucket type="cloudtrail">
-    <name>my-bucket</name>
-    <access_key>insert_access_key</access_key>
-    <secret_key>insert_secret_key</secret_key>
-  </bucket>
```
Comment (Selutario) [inline source/amazon/services/prerequisites/credentials.rst:200]: According to what I said above, this should remain in order to be merged in a `path` version, so only examples should be removed.

File: source/azure/activity-services/prerequisites/credentials.rst [hunk 2]
```diff
@@ -113,70 +107,4 @@ Regardless of the service or activity to be monitored, the authentication file i
 
 Check the :doc:`azure-logs wodle </user-manual/reference/ossec-conf/wodle-azure-logs>` section from the ossec.conf reference page for more information about the ``<auth_path>`` and other available parameters.
 
-
-Inserting the credentials into the configuration
```
Comment (Selutario) [inline source/azure/activity-services/prerequisites/credentials.rst:117]: Same with this, we will need to remove it in 5.0.0, but we should stop showing examples before.

File: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst
Comment (Selutario) [inline source/user-manual/reference/ossec-conf/wodle-azure-logs.rst:1]: Only examples should be removed for now. If you prefer, you can open a new issue for it and assign this PR to the new issue. Then, create a PR for the original issue (https://github.com/wazuh/wazuh-documentation/issues/6363) only for removing examples.

**Review by Selutario — CHANGES_REQUESTED** [review]
The release type for this issue is `patch`, in this PR we should only remove the examples since it makes no sense to show examples of something that we do not recommend using.

Removing all mentions to credentials in configuration could be done in a different PR in the future.

File: source/cloud-security/azure/activity-services/prerequisites/credentials.rst [hunk 3]
```diff
@@ -35,14 +35,8 @@ Getting access credentials for Storage
...
-Authentication options
-----------------------
-
-There are two different ways to set up the Azure authentication:
-
 Using an authentication file
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----------------------------
 
 It is possible to store the credentials in a file for authentication as long as the file content follows the `field = value` format explained below.
```
Comment (GGP1) [inline source/cloud-security/azure/activity-services/prerequisites/credentials.rst:41]: Now that the `Authentication options` title was removed, this subtitle lacks some context. I would suggest doing something like

```
Authentication
---------------

To authenticate, store the credentials in a file whose content follows the `field = value` format explained below.
```

**Review by GGP1 — CHANGES_REQUESTED** [review]

**Review by GGP1 — APPROVED** [review]
LGTM

**Review by Selutario — APPROVED** [review]

File: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst [hunk 4]
```diff
@@ -248,52 +230,24 @@ Defines the use of the Azure Log Analytics REST API to get the desired logs.
...
-| **Default value**  | N/A                |
-+--------------------+--------------------+
-| **Allowed values** | Any string         |
-+--------------------+--------------------+
-
 log_analytics\\auth_path
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Path of the file that contains the application identifier and the application key for authentication in order to use the Azure Log Analytics API. Incompatible with ``application_id`` and ``application_key`` options.
+Path of the file that contains the application identifier and the application key for authentication in order to use the Azure Log Analytics API.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/wodle-azure-logs.rst:250]:
```suggestion
File path containing the application identifier and key for authentication. This is necessary to use the Azure Log Analytics API.
```

File: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst [hunk 5]
```diff
@@ -440,52 +394,24 @@ graph
...
-| **Default value**  | N/A                |
-+--------------------+--------------------+
-| **Allowed values** | Any string         |
-+--------------------+--------------------+
-
 graph\\auth_path
 ^^^^^^^^^^^^^^^^
 
-Path of the file that contains the application identifier and the application key for authentication in order to use the Azure Active Directory Graph API. Incompatible with the ``application_id`` and ``application_key`` options. Check the :doc:`credentials </cloud-security/azure/activity-services/prerequisites/credentials>` reference for more information about this topic.
+Path of the file that contains the application identifier and the application key for authentication in order to use the AAD Graph API.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/wodle-azure-logs.rst:414]:
```suggestion
File path containing the application identifier and key for authentication. This is necessary to use the AAD Graph API.
```

File: source/user-manual/reference/ossec-conf/wodle-azure-logs.rst [hunk 6]
```diff
@@ -603,52 +529,24 @@ storage
...
-| **Default value**  | N/A                |
-+--------------------+--------------------+
-| **Allowed values** | Any string         |
-+--------------------+--------------------+
-
 storage\\auth_path
 ^^^^^^^^^^^^^^^^^^
 
-Path of the file that contains the account name and the account key for authentication. Incompatible with ``account_name`` and ``account_key`` options.
+Path of the file that contains the account name and the account key for authentication.
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/wodle-azure-logs.rst:549]:
```suggestion
File path containing the account name and key for authentication.
```

**Review by javimed — CHANGES_REQUESTED** [review]

**Review by javimed — APPROVED** [review]
LGTM

**Pull request merged**